scripts/rs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "procdockerstatsd_rs"
+version = "1.0.0"
+edition = "2024"
```

Comment:
>2024 [](http://example.com/codeflow?start=11&length=4)

Should it be 2025? minor issue.

Reply:
this is the rusttoml edition, and yea, it is 2024 for now.